lib/node_modules/@stdlib/blas/base/drot/lib/drot.js

```diff
@@ -0,0 +1,89 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MAIN //
+
+/**
+* Applies a plane rotation.
+*
+* @param {PositiveInteger} N - number of indexed elements
+* @param {Float64Array} x - first input array
+* @param {integer} strideX - `x` stride length
+* @param {Float64Array} y - second input array
+* @param {integer} strideY - `y` stride length
+* @param {number} c - double-precision floating-point number
+* @param {number} s - double-precision floating-point number
+* @returns {Float64Array} `y`
+*
+* @example
+* var Float64Array = require( '@stdlib/array/float64' );
+*
+* var x = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0 ] );
+* var y = new Float64Array( [ 6.0, 7.0, 8.0, 9.0, 10.0 ] );
+*
+* drot( x.length, x, 1, y, 1, 0.707, 0.707 );
+* // x => <Float64Array>[ 4.949, 6.3629999999999995, 7.776999999999999, 9.190999999999999, 10.604999999999999 ]
+* // y => <Float64Array>[ 3.535, 3.535, 3.5349999999999997, 3.5349999999999997, 3.5349999999999997 ]
+*/
+function drot( N, x, strideX, y, strideY, c, s ) {
+	var tmp;
+	var ix;
+	var iy;
+	var i;
+	if ( N <= 0 ) {
+		return y;
+	}
+
+	// If both strides are equal to `1`...
+	if ( strideX === 1 && strideY === 1 ) {
+		for ( i = 0; i < N; i++ ) {
+			tmp = ( c * x[ i ] ) + ( s * y[ i ] );
+			y[ i ] = ( c * y[ i ] ) - ( s * x[ i ] );
+			x[ i ] = tmp;
+		}
```

Comment:
```suggestion
		}
		return y;
```

Early return and remove the `else` block, allowing you to reduce indentation of the subsequent loop.